test/runtests.jl

```diff
@@ -14,6 +14,10 @@ end
 #Start Test Script
 
 @time begin
+if GROUP == "All" || GROUP == "Symplectic"
```

Comment:
This is not correct. Could you add the tests into a group that already exists?

Reply:
Yes, you are right. That was to see the results of my changes at first. In the current state this PR must not be merged.